docs/manual/common/guide/production/LightbendPlatform.md

```diff
@@ -0,0 +1,44 @@
+# Using Lightbend Platform with Lagom
+
+Lagom is part of the [Lightbend Platform](https://www.lightbend.com/lightbend-platform) — an operationally robust solution for deploying Reactive Microservices, adding the ability to infuse your applications with intelligence based on real-time streaming and Machine Learning (ML). Lagom services can be developed, deployed and run in production using only open source components, however we recommend considering the advanced features provided by Lightbend Platform.
```

Comment:
the "adding the ability" phrase doesn't strike my ear correctly. How about putting a period after Reactive microservices and starting the next sentence "It (or Lightbend Platform" gives you the ability to infuse applications with intelligence ..."? Also, I think you need another comma after however.

Reply:
I copied the first sentence from https://www.lightbend.com/lightbend-platform, so we should consider changing it there too. CC: @rasummer

Do you mean commas both before and after "however"? That text was in the original Enterprise Suite page, but I'll fix it while I'm in here. Now I've gone down a Google search hole about the correct usage and it seems to suggest semicolon before and comma after for this type of usage: http://www.grammarerrors.com/punctuation/commas-with-conjunctive-adverbs-however-furthermore-etc/ Maybe it's better to split this sentence, too? Alternatively, change "however" to "but" or "although"?

Reply:
Probably easier to parse if you split the sentences.

Reply:
Thanks!